src/server/messages.ts

```diff
@@ -48,40 +50,41 @@ export function validate(filename: string, project: string, text: string): Serve
     }
     const req = request(options);
     req.on('response', (res) => {
-        console.log("DONE");
+        log('info', "DONE");
 
         // DEBUG
-        console.log(`STATUS: ${res.statusCode}`);
-        console.log(`HEADERS: ${JSON.stringify(res.headers)}`);
+        log('debug', `STATUS: ${res.statusCode}`);
+        log('debug', `HEADERS: ${JSON.stringify(res.headers)}`);
         res.setEncoding('utf8');
         if (res.statusCode !== 200) {
-            console.error(`Request failed! Status: ${res.statusCode}`);
+            //log('warn', `Request failed! Status: ${res.statusCode}`);
+            vscode.window.showWarningMessage(`Request failed! Status: ${res.statusCode}`);
```

Comment:
@ageorgou this is what you suggested to do, right?  But probably as an error, not warning?